smartclient-core/src/test/java/org/srg/smartclient/JDBCHandlerUpdateTest.java

```diff
@@ -75,6 +75,63 @@ public void simpleUpdate() throws Exception {
                 }""", response);
     }
 
+    @Test
+    public void simpleUpdateOfMultipleValues() throws Exception {
+        withExtraFields(ExtraField.Email);
+
+        final DSRequest request = JsonTestSupport.fromJSON(new TypeReference<>(){}, """
+               {
+                 useStrictJSON : true,
+                 dataSource : "EmployeeDS",
+                 operationType : "UPDATE",
+                 componentId : "isc_ListGrid_5",
```

Comment:
componentId, startRow,  endRow , textMatchStyle are not requires, please remove them